crates/registry/src/package_version.rs

```diff
@@ -30,13 +29,18 @@ impl PackageVersion {
         http_client: &reqwest::Client,
         registry: &str,
     ) -> Result<Self, RegistryError> {
+        let url = || format!("{0}{name}/{version}", &registry); // TODO: use reqwest url type
```

Comment:
Why do we need this todo?

Reply:
The current type of `registry` is `&str`, and it only works correctly when it ends with a `/`. The `Url` type is more efficient and less error-prone.